lib/test.js

```diff
@@ -8,6 +8,14 @@ const assert = require('./assert');
 const nowAndTimers = require('./now-and-timers');
 const concordanceOptions = require('./concordance-options').default;
 
+function isExternalAssertError(error) {
+	if (error.matcherResult) {
+		return error.matcherResult.actual && error.matcherResult.expected;
+	}
+
+	return error.actual && error.expected;
```

Comment:
Could you add comments clarifying which errors from which frameworks you're looking for here?

Reply:
Absolutely!

Reply:
Nah those can be added in a future PR if necessary.